modules/aws/ec2_test.go

```diff
@@ -0,0 +1,114 @@
+package aws
+
+import (
+	"terratest/modules/packer"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestAmiIsPublic checks whether AMI is publicly accessible
+func TestAmiIsPublic(t *testing.T) {
+	t.Parallel()
+
+	awsRegion := "eu-west-2"
+	packerVars := map[string]string{
+		"aws_region": awsRegion,
+	}
+
+	amiID := BuildAmi(t, packerVars)
+	defer DeleteAmi(t, awsRegion, amiID)
+
+	ec2Client := NewEc2Client(t, awsRegion)
+	MakeAmiPublic(t, amiID, ec2Client)
+	amiIsPublic := GetAmiPubliclyAccessible(t, awsRegion, amiID)
+	assert.True(t, amiIsPublic)
+}
+
+// TestAmiIsPrivateAndNotSharedWithAccount checks whether AMI is private and not shared with any other account
+func TestAmiIsPrivateAndNotSharedWithAccount(t *testing.T) {
+	t.Parallel()
+
+	requestingAccount := "123456789012"
+	awsRegion := "eu-west-1"
+	packerVars := map[string]string{
+		"aws_region": awsRegion,
+	}
+
+	amiID := BuildAmi(t, packerVars)
+	defer DeleteAmi(t, awsRegion, amiID)
+
+	ec2Client := NewEc2Client(t, awsRegion)
+	ShareAmi(t, amiID, "099720109477", ec2Client)
```

Comment:
What is `099720109477`?

Reply:
have switched to using the aws.CanonicalAccountId constant. it was the same thing